Info.lua

```diff
@@ -235,6 +235,23 @@ g_PluginInfo =
 			},
 		},
 
+		["/version"] = 
+		{
+			Alias = "/ver",
+			Permission = "core.version",
+			Handler = HandleVersionCommand,
+			HelpString = "Shows the version of the server.",
+		},
+
+		["/about"] = 
+		{
+			Alias = "/a",
```

Comment:
This seems too generic, unless other server software use it as an alias?

Reply:
SpigotMC, PaperSpigit and CraftBukkit use the same alias name.

Reply:
I still decided to change it to /abt